cmd/commands/permissions.go

```diff
@@ -64,3 +87,54 @@ func resetPermissionsCmdF(command *cobra.Command, args []string) error {
 
 	return nil
 }
+
+type permissionsExportWriter struct{}
+
+func (pew permissionsExportWriter) Write(p []byte) (int, error) {
```

Comment:
Why not use directly os.Stdout?

Reply:
Actually, if the fucntion receive an io.Writer and is expected to produce a JSONL, that is not true, because if I use the function, to export directly to a file, it will a bunch of jsons, all in the same line.

Reply:
@jespino For testability mainly where I would otherwise be testing the output of stdout [here](https://github.com/mattermost/mattermost-server/pull/8787/files#diff-59552339387773049e2fe84ebf62a251R35). Perhaps you're right that the newline character should be present in the output though.

Reply:
@jespino I moved the newline character into the bytes written. Good call.